python2verilog/simulation/iverilog.py

```diff
@@ -66,17 +67,19 @@ def _run_cmd_with_fifos(
 
         try:
             logging.debug("Waiting on process for %ss", timeout)
+            start_time = time.time()
             process.wait(timeout=timeout)
+            logging.debug("Took %ss", time.time() - start_time)
```

Comment:
Is this a hack to wait timeout seconds or smth?

Reply:
No. It's for letting a debugger know how long the process actually takes. So, if they want to make the timeout proportional to a value, they can more easily find a formula.